examples/schema_site_external_epg/main.tf

```diff
@@ -14,103 +14,63 @@ provider "mso" {
   platform = "nd"
 }
 
-resource "mso_site" "test_site" {
-  name         = "test_site"
-  username     = "" # <APIC username>
-  password     = "" # <APIC pwd>
-  apic_site_id = "105"
-  urls         = "" # <APIC site url>
-  location = {
-    lat  = 78.946
-    long = 95.623
-  }
+data "mso_tenant" "demo_tenant" {
```

Comment:
Did we decide to use the data source instead of creating the resource in the examples?
The only issue would be inconsistency across all the examples.

Reply:
Data source is not an issue on this case as it is not centered around tenant or site.